.ci/static-checks.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Copyright (c) 2017 Intel Corporation                                                                                                                                                        
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Description: Central script to run all static checks.
+#   This script should be called by all other repositories to ensure 
+#   there is only a single source of all static checks.
+
+set -e
+
+test_repo="github.com/clearcontainers/tests"
+
+# Clone Tests repository.
+go get "$test_repo"
```

Comment:
don't know if you need any magic options on the `go get` here, maybe `-u` or `-d` and some judicious `...`'s ? - just asking, as I know we've hit such 'non-buildable repos issue warnings' situations before.